src/components/Card/CartCard.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { useCart } from "../../context/cart-context";
```

Comment:
Try barrel export and try to make use of absolute imports because it makes your code looks clean.